README.md

```diff
@@ -96,7 +90,7 @@ to improve it:
 [contrib guide]: https://www.ngraph.ai/documentation/contributing/guide
 [pull request]: https://github.com/NervanaSystems/ngraph/pulls
 [how to import]: https://www.ngraph.ai/tutorials/onnx-tutorial#import-a-model-with-onnx-and-ngraph
-[ngraph_wireframes_with_notice]: doc/sphinx/source/graphics/ngpipelines.png "nGraph wireframe"
+[ngraph_wireframes_with_notice]: doc/sphinx/source/graphics/nGraph_main.png "nGraph components as of Sept 2019"
```

Comment:
@indie - Why do you suggest we add the date?

Reply:
What we refer to as "Beta" has changed drastically since March... this tag doesn't actually show up anywhere in the rendered doc (pretty sure); it's just there for us to have a barometer on how recently we've updated the components illustration.